src/archivematicaCommon/lib/elasticSearchFunctions.py

```diff
@@ -583,6 +597,34 @@ def normalize_list_dict_elements(data):
             data[index] =  normalize_dict_values(value)
     return data
 
+def _get_file_formats(f):
+    formats = []
+    fields = ['format_version__pronom_id',
+              'format_version__description',
+              'format_version__format__group__description']
+    for puid, format, group in f.fileformatversion_set.all().values_list(*fields):
+        formats.append({
+            'puid': puid,
+            'format': format,
+            'group': group,
+        })
+
+    return formats
+
+def _list_bulk_extractor_reports(transfer_path, file_uuid):
+    reports = []
+    log_path = os.path.join(transfer_path, 'logs', 'bulk-' + file_uuid)
```

Comment:
Is there a way to get this info without assuming the backlogged transfer is locally accessible?  In the future, it would be nice if it could be in Duracloud or something.  Does the storage service browse endpoint work here?